bluebrain/repo-bluebrain/packages/neurodamus-models/package.py

```diff
@@ -73,9 +73,16 @@ def setup_run_environment(self, env):
         env.set("HOC_LIBRARY_PATH", self.prefix.share.join(f"neurodamus_{mech}").hoc)
         if "+coreneuron" in self.spec:
             env.set("CORENEURONLIB", self.prefix.lib + "/libcorenrnmech.so")
```

Comment:
Could you also adapt the setting for `CORENEURONLIB` ?  It is `libcorenrnmech.dylib` for macOs as well.

Reply:
ah thank you for telling me. Yes, ofc!